README.md

```diff
@@ -29,14 +29,30 @@ You also need to install the dependencies required by React Native.
 Go to the [React Native environment setup](https://reactnative.dev/docs/environment-setup), then select `React Native CLI Quickstart` tab.  
 Follow instructions for your given `development OS` and `target OS`.
 
-## Quick Start
+## Getting Started
+
+- Prerequisites
+
+  - Node.js and yarn should be installed on your system.
+  - Git should be installed on your system.
+  - [React Native CLI should be installed on your system.](https://reactnative.dev/docs/environment-setup)
```

Comment:
These info are very identical to above `Requirement` section, I think we can remove it

Reply:
Done.